Dockerfile

```diff
@@ -35,5 +37,5 @@ RUN rm -rf /root/.lein /root/.m2 /root/.node-gyp /root/.npm /root/.yarn /root/.y
 EXPOSE 3000
 
 # build and then run it
-WORKDIR /app/source
+WORKDIR /app
```

Comment:
Won't `./bin/start` no longer exist since you're changing the workdir away from `/app/source/`?